integration/v3_grpc_test.go

```diff
@@ -432,3 +432,205 @@ func TestV3WatchFromCurrentRevision(t *testing.T) {
 		clus.Terminate(t)
 	}
 }
+
+// TestV3WatchCancel tests Watch APIs cancellation.
+func TestV3WatchCancel(t *testing.T) {
+	tests := []struct { // send Put, WatchRequest, and then receive WatchResponse in order
+		preqs []*pb.PutRequest
+		wreqs []*pb.WatchRequest
```

Comment:
let's change this to watchReqs or it is confusing since wX in tests means wantedX.